File: AkitaModel/Flagging/ResultTypes.cs
namespace Skyware.Lis.AkitaModel.Flagging;

/// <summary>
/// Result types.
/// </summary>
public enum ResultTypes : byte
{
    /// <summary>
    /// Textual result.
    /// </summary>
    Text = 4,
    
    /// <summary>
    /// Numeric result, usually represents concentration.
    /// </summary>
    Quantitative = 1,

    /// <summary>
    /// Semi-quantitative, cut-off result.
    /// </summary>
    SemiQuantitative = 5,

    /// <summary>
    /// Ratio, such as 1:2, 2:16, 3:5, etc.
    /// </summary>
    Ratio = 2,

    /// <summary>
    /// Value amongst list of allowed values.
    /// </summary>
    List = 3,

}

File: AkitaMotelTS/src/ResultTypes.ts
/**
 * Skyware.Lis.AkitaModel
 */

/**
 * Result types.
 */
export enum ResultTypes {
  /**
   * Textual result.
   */
  Text = 4,

  /**
   * Numeric result, usually represents concentration.
   */
  Quantitative = 1,

  /**
   * Semi-quantitative, cut-off result.
   */
  SemiQuantitative = 5,

  /**
   * Ratio, such as 1:2, 2:16, 3:5, etc.
   */
  Ratio = 2,

  /**
   * Value amongst list of allowed values.
   */
  List = 3
}
